tfctl/version.go

```diff
@@ -0,0 +1,47 @@
+package tfctl
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Version prints the installed version of tf-controller and the tfctl cli
+func (c *CLI) Version(out io.Writer) error {
+	deployment := &appsv1.Deployment{}
+	if err := c.client.Get(context.TODO(), types.NamespacedName{
+		Namespace: c.namespace,
+		Name:      "tf-controller",
+	}, deployment); client.IgnoreNotFound(err) != nil {
+		return err
+	}
+
+	var (
+		version       string
+		runnerVersion string
+	)
+
+	if len(deployment.Spec.Template.Spec.Containers) > 0 {
+		for _, c := range deployment.Spec.Template.Spec.Containers {
+			if c.Name == "manager" {
+				version = strings.Split(c.Image, ":")[1]
```

Comment:
Given that it's possible to have an image with no `:` here, this would give you an ugly error.

Simple version, check the number of splits, more complex version, use [name.ParseReference](https://pkg.go.dev/github.com/google/go-containerregistry@v0.8.0/pkg/name#ParseReference) to parse the string. 

Personally, I'd just count the split :-)